src/calculation/ClauseResultsBuilder.ts

```diff
@@ -363,100 +371,149 @@ export function prettyResult(result: any | null, indentLevel?: number, keyIndent
   if (result instanceof DateTime) {
     return moment.utc(result.toString()).format('MM/DD/YYYY h:mm A');
   } else if (result instanceof Interval) {
-    return `INTERVAL: ${prettyResult(result.low)} - ${prettyResult(result.high)}`;
+    const typeStr = includeType ? 'INTERVAL: ' : '';
+    return `${typeStr}${prettyResult(result.low)} - ${prettyResult(result.high)}`;
   } else if (result instanceof Code) {
-    // TODO: Sort out a better way to have a friendly system display for FHIR codes
-    return `CODE: ${result.system} ${result.code}`;
+    return prettyCode(result.code, result.system, result.display, includeType);
+  } else if (result instanceof Concept) {
+    return prettyConcept(result.display, result.codes, includeType, indentLevel, keyIndent);
```

Comment:
One thing I noticed that I am not even sure if it's a problem or not is that the pretty result for SDE Sex and SDE Race are different (one is in square brackets and the other is not). I believe they are both Coding types, so should they be the same? I did some digging but I got stuck but it looks like the SDE race statement goes through the `prettyFHIRObject` function where the SDE sex statement only goes through the `prettyCode` function.

![Screenshot 2023-06-27 at 11 38 38 AM](https://github.com/projecttacoma/fqm-execution/assets/30158156/ca69fae0-afe7-4326-9935-1280b4bc0d07)

Reply:
I believe these statements return different data types. SDE Race expression (in the base FHIR version for colorectal cancer screening) returns a list of FHIR Codings. SDE Sex returns a singular CQL system Code